docs/app/partials/layout-introduction.tmpl.html

```diff
@@ -0,0 +1,364 @@
+    </tr>
+    <tr>
+      <td>layout-align</td>
+      <td><code>start|center|end|space-around|space-between</code> <code>start|center|end|stretch</code></td>
+    </tr>
+    <tr>
+      <td>layout-fill</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>layout-wrap</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>layout-nowrap</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>layout-margin</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>layout-padding</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>show</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>hide</td>
+      <td></td>
+    </tr>
+    </tbody>
+  </table>
+
+
+  <p>And if we use Breakpoints as specified in Material Design:</p>
+  <p><a
+      href="https://camo.githubusercontent.com/ad81ae92f8b4285747ce4e48007bf3f104dd5630/687474703a2f2f6d6174657269616c2d64657369676e2e73746f726167652e676f6f676c65617069732e636f6d2f7075626c6973682f6d6174657269616c5f765f342f6d6174657269616c5f6578745f7075626c6973682f3042386f6c5631354a3761625053474678656d46695156527462316b2f6c61796f75745f61646170746976655f627265616b706f696e74735f30312e706e67"
+      target="_blank"><img
+      src="https://camo.githubusercontent.com/ad81ae92f8b4285747ce4e48007bf3f104dd5630/687474703a2f2f6d6174657269616c2d64657369676e2e73746f726167652e676f6f676c65617069732e636f6d2f7075626c6973682f6d6174657269616c5f765f342f6d6174657269616c5f6578745f7075626c6973682f3042386f6c5631354a3761625053474678656d46695156527462316b2f6c61796f75745f61646170746976655f627265616b706f696e74735f30312e706e67"
+      alt=""
+      style="max-width:100%;"></a>
+  </p>
+
+
+  <p>We can associate breakpoints with mediaQuery definitions using breakpoint <strong>alias(es)</strong>:</p>
+
+  <table class="md-api-table">
+    <thead>
+    <tr>
+      <th>Breakpoint</th>
+      <th>MediaQuery (pixel range)</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td>xs</td>
+      <td>'(max-width: <b>599</b>px)'</td>
+    </tr>
+    <tr>
+      <td>gt-xs</td>
+      <td>'(min-width: <b>600</b>px)'</td>
+    </tr>
+    <tr>
+      <td>sm</td>
+      <td>'(min-width: <b>600</b>px) and (max-width: <b>959</b>px)'</td>
+    </tr>
+    <tr>
+      <td>gt-sm</td>
+      <td>'(min-width: <b>960</b>px)'</td>
+    </tr>
+    <tr>
+      <td>md</td>
+      <td>'(min-width: <b>960</b>px) and (max-width: <b>1279</b>px)'</td>
+    </tr>
+    <tr>
+      <td>gt-md</td>
+      <td>'(min-width: <b>1280</b>px)'</td>
+    </tr>
+    <tr>
+      <td>lg</td>
+      <td>'(min-width: <b>1280</b>px) and (max-width: <b>1919</b>px)'</td>
+    </tr>
+    <tr>
+      <td>gt-lg</td>
+      <td>'(min-width: <b>1920</b>px)'</td>
+    </tr>
+    <tr>
+      <td>xl</td>
+      <td>'(min-width: <b>1920</b>px)'</td>
+    </tr>
+    </tbody>
+  </table>
+
+  <br/>
+  <hr>
+  <h3>
+    API with Responsive Breakpoints
+  </h3>
+
+  <p>Now we can combine the breakpoint <code>alias</code> with the Layout API to easily support Responsive breakpoints
```

Comment:
This section is worded like you're in the middle of a step-by-step tutorial rather than API docs. Instead something like
"The breakpoint alias combines with the Layout API, allowing..."